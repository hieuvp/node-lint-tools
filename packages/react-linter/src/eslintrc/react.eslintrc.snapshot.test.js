const react = require('./react.eslintrc');

it('should match snapshot', () => {
  expect(react).toMatchSnapshot();
});
