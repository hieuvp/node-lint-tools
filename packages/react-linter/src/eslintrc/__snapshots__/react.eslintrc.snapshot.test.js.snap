// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "extends": Array [
    "airbnb-base",
    "plugin:jest/recommended",
    "plugin:jsdoc/recommended",
    "plugin:node/recommended",
    "plugin:prettier/recommended",
    "plugin:promise/recommended",
    "plugin:security/recommended",
    "airbnb",
    "plugin:import/recommended",
    "plugin:jsx-a11y/recommended",
    "plugin:react/recommended",
    "prettier/react",
  ],
  "plugins": Array [
    "jest",
    "jsdoc",
    "node",
    "optimize-regex",
    "prettier",
    "promise",
    "security",
    "import",
    "jsx-a11y",
    "react",
  ],
  "rules": Object {
    "comma-dangle": Array [
      "error",
      "never",
    ],
    "import/no-extraneous-dependencies": Array [
      "error",
      Object {
        "devDependencies": Array [
          "/runfile.js",
          "/src/**/*.test.js",
          "/src/**/*.spec.js",
          "/packages/**/src/**/*.test.js",
          "/packages/**/src/**/*.spec.js",
        ],
      },
    ],
    "jsdoc/require-param-description": "off",
    "optimize-regex/optimize-regex": "warn",
    "prettier/prettier": Array [
      "error",
      Object {
        "printWidth": 90,
        "singleQuote": true,
      },
    ],
  },
}
`;
